src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -39,6 +39,14 @@ const Body = BaseForm.extend(Object.assign(
         </div>`);
     },
 
+    getUISchema () {
+      // Change the UI schema to not display radios here.
+      const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
```

Comment:
can simple return empty as we didn't render anything from ui schema ?

Reply:
Two reasons - 
* I didn't want to mess with any future changes, the schema may have more fields to be displayed.
* Currently there's another entry with `authenticator.id`, I just let it be as is..

Reply:
https://oktainc.atlassian.net/browse/OKTA-312428

Reply:
- yeah probably need to keep `authenticatorId`.
- `any future changes, the schema may have more fields to be displayed. ` this is ideal. we likely need adjust UI when any more fields.
- Would prefer to use `filter` instead of `findIndex` and `splice`. ?

```
return uiSchemas.filter( where name !== 'authenticator.methodType' )
```

Reply:
Sure, that looks better to me.